src/openshift/cluster.ts

```diff
@@ -66,17 +66,19 @@ export class Cluster extends OpenShiftItem {
         Cli.getInstance().showOutputChannel();
     }
 
-    static async openshiftConsole(context: OpenShiftObject): Promise<void> {
-        if (context) {
-            open(`${context.getName()}/console`);
+    static async openshiftConsole(): Promise<void> {
+        let consoleUrl: string;
+        const versionInfo = await Cluster.odo.execute(Command.clusterVersionDescribe(), process.cwd(), false);
```

Comment:
Why is it required to call `oc describe` first?

Reply:
Now to get to know which version of server is used, either 3.x or 4.x, there is no way from oc or odo.
The ``oc describe clusterversion` command works for 4.x server and errors out for 3.x. So the check is done for that.

For 3.x, there is no `clusterversion`.